src/main/java/org/jabref/logic/cleanup/RelativePathsCleanup.java

```diff
@@ -57,4 +64,21 @@ public RelativePathsCleanup(BibDatabaseContext databaseContext, FilePreferences
         return Collections.emptyList();
     }
 
+    /**
+     * Checks, if the given file path is a well-formed web url
+     *
+     * @param filePath file path to check
+     * @return <code>true</code>, if the given file path is a web url, <code>false</code> otherwise
+     */
+    private static boolean isWebUrl(String filePath)
```

Comment:
The class `LinkedFile` already contains a method that does this job:
https://github.com/JabRef/jabref/blob/8520833e684aa9e305dd8745e45696ffd6d984fb/src/main/java/org/jabref/model/entry/LinkedFile.java#L162-L164
Can you please use it. I would also suggest to add the normalization `trim().toLowerCase()` there, which I find a nice addition.

Reply:
Concerning isOnlineLink(): I would recommend removing "toCheck.contains("www.")", because a path to a local file might contain this pattern coincidentally. What do you say?